project/BuildSettings.scala

```diff
@@ -67,7 +67,10 @@ object BuildSettings {
 
   lazy val awsSettings = appSettings ++ Seq(
     name := "lake-loader-aws",
-    buildInfoKeys += BuildInfoKey("cloud" -> "AWS")
+    buildInfoKeys += BuildInfoKey("cloud" -> "AWS"),
+
+    // TODO: Remove this after Hadoop 3.5.0 is released with full support for V2 SDK
```

Comment:
We might want to put a comment next to Hadoop version as well

Reply:
I absolutely promise I will remember :)  This Hadoop release (+ removing aws sdk 1) is a big thing for Lake Loader, and I am following it closely.